benchmark.go

```diff
@@ -112,13 +142,6 @@ func runBenchmark(out output, logger *zap.Logger, allOpts Options, resolved reso
 
 	goMaxProcs := opts.setGoMaxProcs()
 	numConns := opts.getNumConnections(goMaxProcs)
-	out.Printf("Benchmark parameters:\n")
```

Comment:
the parameters are printed before we started the benchmark in the current state -- can we retain that?

for JSON, it's a little trickier, we may have to defer it to the end.

Reply:
Changed to print the parameters before starting benchmark for plaintext.